icubam/backoffice/server.py

```diff
@@ -80,37 +116,44 @@ def make_routes(self, path):
 
     if os.path.isdir(self.config.backoffice.extra_plots_dir):
       route = os.path.join("/", self.root, r'static/extra-plots/(.*)')
-      self.routes.append(
-          (route, tornado.web.StaticFileHandler,
-          {'path': self.config.backoffice.extra_plots_dir})
-          )
+      self.routes.append((
+        route, tornado.web.StaticFileHandler, {
+          'path': self.config.backoffice.extra_plots_dir
+        }
+      ))
```

Comment:
Irrelevant formatting issues after running `yapf`. We should add it to CI.

Reply:
@rth OK I can get on it